app/controllers/departments_controller.rb

```diff
@@ -50,24 +51,7 @@ def set_department
     @department = Department.find(params[:id])
   end
 
-  def phone_attrs
-    [:id, :body, :label, :_destroy, :type, :contacts_id]
-  end
-
-  def email_attrs
-    [:id, :body, :label, :_destroy, :type, :contacts_id]
-  end
-
-  def contact_attrs
-    [
-      :id, :name, :_destroy, :contactable_id, :contactable_type, :street,
-      :extended, :city, :postal_code,
-      contact_emails_attributes: email_attrs,
-      contact_phones_attributes: phone_attrs
-    ]
-  end
-
   def department_params
-    params.require(:department).permit(user_ids: [], contact_attributes: contact_attrs)
+    params.require(:department).permit(policy(@department).permitted_attributes)
```

Comment:
PS: To demonstrate this I've just added a comment to https://aoz-003-staging.panter.biz/clients/7 as a socialworker. This action whould simply destroy all the text that is in there, if any, and place the text of my choice in there.